tensorflow/python/keras/activations.py

```diff
@@ -143,22 +143,38 @@ def softsign(x):
 @keras_export('keras.activations.relu')
 def relu(x, alpha=0., max_value=None, threshold=0):
   """Rectified Linear Unit.
+      the ReLU function is non linear, which means we can easily
+      backpropagate the errors and have multiple layers of neurons being
+      activated by the ReLU function.The main advantage of using the ReLU
+      function over other activation functions is that it does not activate
+      all the neurons at the same time.
 
-  With default values, it returns element-wise `max(x, 0)`.
+  With default values, it returns element-wise maximum of input tensor and 0.
 
   Otherwise, it follows:
   `f(x) = max_value` for `x >= max_value`,
   `f(x) = x` for `threshold <= x < max_value`,
   `f(x) = alpha * (x - threshold)` otherwise.
 
   Arguments:
-      x: A tensor or variable.
-      alpha: A scalar, slope of negative section (default=`0.`).
-      max_value: float. Saturation threshold.
-      threshold: float. Threshold value for thresholded activation.
+      `x`: A tensor or variable.
+      `alpha`: A float, governs as the slope of function when the input value is less than threshold value, small positive value corresponds to `Leaky RelU` (default=`0.`).
+      `max_value`: float. The maximum value the function could return.(default=`none`)
+      `threshold`: float. Threshold value for thresholded activation.
 
   Returns:
-      A tensor.
+      f(x), element-wise. It is a tensor if x is a tensor.
+
+  Example:
+      `python
+      Code:
+      a = tf.constant([1.0, -0.5, 3.4, 2.1, 0.0, -6.5], dtype = tf.float32)
+      b=tf.keras.activations.relu(a,alpha=1.0,max_value=3.0,threshold=1.2)
+      print ("Output: ", (b))
+
+      Output:
+      tf.Tensor([-0.20000005 -1.7         3.          2.1        -1.2
```

Comment:
We can pretty-print this and reduce the number of digits for -0.2.

We also probably want a more realistic example with alpha=0 and threshold=0 as that's what most people want